lib/src/Gren.js

```diff
@@ -913,6 +914,27 @@ class Gren {
         );
     }
 
+    /**
+     * Check that the require tags are exists in tags
+     *
+     * @param {Array} tags
+     * @param {any} requireTags
```

Comment:
You can expect `requireTags` to be `Array` because `this.options.tags` it's always an `Array`

https://github.com/github-tools/github-release-notes/blob/3e6b2e40e20a1abca065bfacad58d25164c1d1db/lib/src/_utils.js#L123-L149

Also, replace the `@return` with [`@throws`](http://usejsdoc.org/tags-throws.html) as the function is only returning `undefined`